hpedockerplugin/volume_manager.py

```diff
@@ -909,6 +917,9 @@ def _get_snapshot_response(self, snapinfo, snapname):
         if 'snap_schedule' in metadata:
             snap_detail['snap_schedule'] = metadata['snap_schedule']
 
+        if '3par_vol_name' in snapinfo:
+            snap_detail['3par_vol_name'] = snapinfo.get('3par_vol_name')
```

Comment:
@wdurairaj inspect will not show backend snapname for older volumes.

Reply:
@nilangekarss , This change will not be applicable for 2.x created volumes. That's my assumption as well. Doing a trace of the volume name and updating etcd as part of inspect is a costly operation, which I would like to avoid.